rel/i18n/emqx_schema.hocon

```diff
@@ -1507,6 +1507,9 @@ Advantages of Disabling<br/>
 - Reduced Memory Usage: Turning off the session registry can lower the overall memory footprint of the system.
 - Improved Performance: Without the overhead of maintaining a global registry, the node can process client connections faster."""
 
+broker_enable_linear_session_registry.desc:
+"""Enable Linearizable Session Registry for correctness."""
```

Comment:
Please elaborate what this feature does. Perhaps rehash the contents of changes/ee/feat-15064.en.md ?